.env

```diff
@@ -1,6 +1,8 @@
 # DOMAIN such as example.com or local development 192.168.1.2
-DOMAIN=192.168.1.30
+DOMAIN=localhost
 PORT=5006
+EMAIL=
+CERT_PATH=./data/certbot
 TZ=Asia/Tokyo
 USE_PROXY="True"
```

Comment:
For this TZ=Asia/Tokyo  could you explain about the generic?@@

Reply:
Using the system default it's running on? I don't know if that's possible.

Reply:
Thank you so much! I would like to find out a way to do so.

Reply:
Solved by mounting /etc/localtime